### PR TITLE
Fix/email message payment alert

### PR DESCRIPTION
**Pull request by varun257**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #DTRSD-50951
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

fix translation issue for alert
add pre wrap and word break to remove horizontal scroll

### CODE REVIEW

**Comment by varun257** [comment]
@antleblanc,
I rebased with latest of develop but i keep getting
`error Couldn't find package "@ovh-ux/manager-billing@^0.5.0-alpha.0 || ^0.6.0-alpha.0" required by "@ovh-ux/manager-dedicated@17.6.0" on the "npm" registry.`
How do i fix this ?

File: packages/manager/modules/billing/src/translations/Messages_en_GB.json [hunk 1]
```diff
@@ -180,7 +180,7 @@
   "ovhAccount_create_alert": "Set up an alert",
   "ovhAccount_create_alert_resume": "You will be alerted by e-mail when your account balance falls below the alert threshold.",
   "ovhAccount_delete_alert": "Disable balance alerts on your account:",
-  "ovhAccount_create_alert_amount": "Altert threshold amount ",
+  "ovhAccount_create_alert_amount": "Alert threshold amount ",
```
Comment (antleblanc) [inline packages/manager/modules/billing/src/translations/Messages_en_GB.json:183]: **issue:** Must be fixed via a translation request.

**Review by antleblanc — CHANGES_REQUESTED** [review]

**Comment by antleblanc** [comment]
> @antleblanc, I rebased with latest of develop but i keep getting `error Couldn't find package "@ovh-ux/manager-billing@^0.5.0-alpha.0 || ^0.6.0-alpha.0" required by "@ovh-ux/manager-dedicated@17.6.0" on the "npm" registry.` How do i fix this ?

There's an issue with following commit: e58ac8b98ed009aa93151f0ae4d52e31f64cfdce

Commit message used is a mixed between commit type feat and fix. Is it normal?

This commit introduced manual changes to given file `Messages_en_GB.json`.
According to our CONTRIBUTING.md file, this is not allowed.
Please prefer using a Translation Request to do so.

If you are introducing a `feat` for the given module named `@ovh-ux/manager-billing`,
since it's in version `0.6.0-alpha.1`, it means that according to our release
script and the `.prerelease` file, you'll get `0.7.0-alpha`.

Since this package is a private one, you'll have to update wherever the package
is used, the SemVer range in order to be compatible with.

File: packages/manager/modules/billing/src/translations/Messages_en_GB.json (hunk 1 above)
Comment (varun257) [reply packages/manager/modules/billing/src/translations/Messages_en_GB.json:183]: Translation request was raised and has been resolved here: https://projects.dsi.ovh/browse/TRANS-39056.
So this has been added in the entry so this should be good ?

Comment (antleblanc) [reply packages/manager/modules/billing/src/translations/Messages_en_GB.json:183]: Yes it is!

I was confused due to the commit message.
Translations used to be retrieved via a given commit message pattern which is not the case here (maybe due to a squash/rebase)

> fix(i18n): add missing translations [CDS XXXX]

**Review by antleblanc — APPROVED** [review]
Changes looks good to me.
Don't forget to update Pull Request description.
Thank you!

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**